app/graphql/types/query_type.rb

```diff
@@ -38,6 +38,25 @@
     end)
   end
 
+  field :search, !Types::SearchResultType do
+    description 'Search Ontohub'
+
+    argument :query, !types.String do
+      description 'The query string'
+    end
+
+    argument :category, Types::SearchResult::CategoryType do
```

Comment:
I was orientating on the [GitHub search](https://github.com/search?q=foobar), where you can see the number of results for all categories, but only see one category of results at a time. That's also the reason for the structure with the `entries` and `count` fields.

That means that when you switch to a different tab, you do a new search request with a different category argument. I'm open for other ideas however, as there are certainly downsides to this.